cli/cmd/common/plugin.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
```

Comment:
let's call this package something descript, I vote `plugin`.

Reply:
+1, I think I originally had that but changed it for some reason. I think it would make more sense though, so will get that switched.